js/main.js

```diff
@@ -19,6 +21,82 @@ function submission(event) {
   formObject.entryId = data.nextEntryId;
   data.nextEntryId++;
   data.entries.unshift(formObject);
-  $image.src = 'images/placeholder-image-square.jpg';
+  $newImage.src = 'images/placeholder-image-square.jpg';
   $form.reset();
+  $ul.prepend(renderEntry(data.entries[0]));
+  toggleNoEntries();
+  viewEntries();
+}
+
+function renderEntry(entry) {
+  var $li = document.createElement('li');
+  $li.setAttribute('class', 'row');
+
+  var $divImg = document.createElement('div');
+  $divImg.setAttribute('class', 'column-half');
+  $li.appendChild($divImg);
+
+  var $img = document.createElement('img');
+  $img.src = entry.imageUrl;
+  var imgEntry = 'img' + entry.entryId;
+  $img.setAttribute('id', imgEntry);
+  $divImg.appendChild($img);
+
+  var $divText = document.createElement('div');
+  $divText.setAttribute('class', 'column-half');
+  $li.appendChild($divText);
+
+  var $h2 = document.createElement('h2');
+  $h2.textContent = entry.title;
+  var h2Entry = 'h2' + entry.entryId;
+  $h2.setAttribute('id', h2Entry);
+  $divText.appendChild($h2);
+
+  var $p = document.createElement('p');
+  $p.textContent = entry.notes;
+  var pEntry = 'p' + entry.entryId;
+  $p.setAttribute('id', pEntry);
+  $divText.appendChild($p);
+  return $li;
+}
+
+document.addEventListener('DOMContentLoaded', getEntry);
+
+function getEntry(entry) {
+  for (var i = 0; i < data.entries.length; i++) {
```

Comment:
You still need to call `toggleNoEntries` in this function, just not in the loop. Also, does this show the last view the user was on when the page loads? Refresh your page on the form view and the entries view, and verify that it reloads on the same view.